packages/x-components/tests/e2e/cucumber/common-steps.spec.ts

```diff
@@ -101,3 +102,11 @@ Given('a suggestions API', () => {
     });
   });
 });
+
+Given('a related tags API', () => {
+  cy.intercept('https://api.empathy.co/getRelatedTags', req => {
+    req.reply({
+      relatedTags: getRelatedTagsStub()
+    });
+  }).as('interceptedRelatedTags');
```

Comment:
Interceptions from the `common-steps` shouldn't need to be checked by the tests as they are generic. No need to make an alias for them.

```suggestion
  });
```

- [ ] Remove alias